test/python/transpiler/test_setlayout.py

```diff
@@ -109,6 +109,13 @@ def test_raise_when_layout_len_does_not_match(self):
         with self.assertRaises(TranspilerError):
             pass_manager.run(circuit)
 
+    def test_raise_if_int_list_layout_contains_duplicates(self):
+        """Test the error is raised if the specified intlist contains duplicates"""
+        circuit = QuantumCircuit(4)
+        layout_pass = SetLayout([0, 1, 1, 2])
+        with self.assertRaises(TranspilerError):
+            layout_pass(circuit)
```

Comment:
I mean, we _did_ just define a new exception type haha

Reply:
Never underestimate the ease of copy and paste :) Updated in: https://github.com/Qiskit/qiskit/pull/11653/commits/4799c86bd64ba98955ccc714b944abd5895ddb0f